src/components/dashboard/Claim/ButtonBlock.js

```diff
@@ -14,18 +14,21 @@ const ButtonBlock = ({
   isInQueue,
   ...props
 }) => {
-  const onPress = useCallback(e => {
-    e.preventDefault()
+  const onPress = useCallback(
```

Comment:
pls use useOnPress hook
use it everywhere if you'll found that .preventDefault is called manually or no preventDefault called.
then don't forgot to apply this change in a separate PR (or add link to the code block you've fixed in the comments for update useOnPress branch later)

Reply:
done